src/core/textrenderer/qgstextrenderer.cpp

```diff
@@ -912,8 +938,8 @@ void QgsTextRenderer::drawBackground( QgsRenderContext &context, QgsTextRenderer
         }
         renderedSymbol.reset( );
 
-        QgsSymbolLayer *symL = QgsSvgMarkerSymbolLayer::create( map );
-        renderedSymbol.reset( new QgsMarkerSymbol( QgsSymbolLayerList() << symL ) );
+        //QgsSymbolLayer *symL = QgsSvgMarkerSymbolLayer::create( map );
+        //renderedSymbol.reset( new QgsMarkerSymbol( QgsSymbolLayerList() << symL ) );
```

Comment:
commented out code

Reply:
Forgot to uncomment, still need to figure out why shadows are weird and blotchy. Especially with justified. Unsure how critital shadows are. Reverting those changes.